ddocs/medic-client/views/docs_by_id_lineage/map.js

```diff
@@ -0,0 +1,20 @@
+function(doc) {
```

Comment:
I'm not sure that it's clearer because the lineage isn't a value, it ends up part of the key. You could query for `key=[ 'abc', 1]` and get the parent of the `abc` doc. Does that make sense?